lib/Twig/Extension/TemplatingHelperExtension.php

```diff
@@ -17,111 +14,31 @@
 
 namespace Pimcore\Twig\Extension;
 
-use Pimcore\Templating\PhpEngine;
-use Symfony\Component\Templating\Helper\HelperInterface;
-use Twig\Extension\AbstractExtension;
-use Twig\TwigFunction;
-
-/**
- * Delegates calls to PHP templating helpers. Use this only with templating helpers which do not rely
- * on PHP rendering!
- *
- * @deprecated
- */
-class TemplatingHelperExtension extends AbstractExtension
-{
-    /**
-     * @var PhpEngine
-     */
-    private $phpEngine;
-
+@trigger_error(
+    sprintf(
+        'Class "%s" is deprecated since v6.9 and will be removed in 7. Use one of these "%s", "%s", "%s", "%s" instead.',
+        TemplatingHelperExtension::class,
+        HeaderExtension::class,
+        PimcoreToolExtension::class,
+        HelpersExtension::class,
+        CacheExtension::class
+    ),
+    E_USER_DEPRECATED
+);
+
+class_exists(HeaderExtension::class);
+class_exists(CacheExtension::class);
+class_exists(PimcoreToolExtension::class);
+class_exists(HelpersExtension::class);
+
+if (false) {
```

Comment:
I don't think so we need this check here, since there is no class alias.

Reply:
please remove class_exists checks, so that TemplatingHelperExtension class is defined and marked as deprecated.